website/docs/r/automation_job_schedule.html.markdown

```diff
@@ -67,5 +71,5 @@ The `timeouts` block allows you to specify [timeouts](https://www.terraform.io/l
 Automation Job Schedules can be imported using the `resource id`, e.g.
 
 ```shell
-terraform import azurerm_automation_job_schedule.example /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/group1/providers/Microsoft.Automation/automationAccounts/account1/jobSchedules/10000000-1001-1001-1001-000000000001
+terraform import azurerm_automation_job_schedule.example /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/group1/providers/Microsoft.Automation/automationAccounts/account1/runBook/book1/schedule/schedule1
```

Comment:
Is there a reason we are creating a new "fake" resource id instead of creating a composite resource id? ie
`/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/group1/providers/Microsoft.Automation/automationAccounts/account1| /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/group1/providers/Microsoft.Automation/automationAccounts/account1/runbooks/Get-AzureVMTutorial`

this is something we are looking to implement first class support for soon and will work with the up coming casing fixes - see https://github.com/hashicorp/go-azure-helpers/pull/208/files

Reply:
Yes, hopefully it'll be merged in the next couple weeks

Reply:
@wuxu92 - the above PR has been merged

Reply:
https://github.com/hashicorp/terraform-provider-azurerm/pull/25446